### DISCUSSION TITLE
Problema scrutinio recupero debiti

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#731] [Richieste di aiuto]
Buonasera quando io schiaccio per impostare l esito per lo scrutinio per il recupero debiti fa un caricamento infinito come posso risolvere?

**Comment by Davide031** [comment]
Confermo che non e un problema di connessione

**Reply by Davide031** [reply]
O di dispositivo

**Comment by trinko** [comment]
Questa non è l'area adatta per le segnalazioni di errore.
Vai alla segnalazione #732
